cmd/content-sources/main.go

```diff
@@ -40,11 +40,20 @@ func main() {
 	config.Load()
 	config.ConfigureLogging()
 	err := db.Connect()
-	defer db.Close()
-
 	if err != nil {
 		log.Fatal().Err(err).Msg("Failed to connect to database.")
 	}
+	defer db.Close()
+
+	if argsContain(args, "api") {
+		err = config.ConfigureTang()
+		if err != nil {
+			log.Panic().Err(err).Msg("Could not initialize tang, was pulp database information provided?")
+		}
+		if config.Tang != nil {
+			defer (*config.Tang).Close()
+		}
+	}
```

Comment:
i wanted to point this out, here i'm only loading tang when running the api.  This prevents us from having to configure the task workers which don't need it.  Curious if there's any thoughts on this

Reply:
makes sense